config/schemas/config.yaml

```diff
@@ -1693,6 +1693,13 @@ properties:
               `RetentionDays` defines how old a backup should be before deleting it.
             default: 7
             type: number
+          schedule:
+            title: Schedule for backup job
+            description: |-
+              `schedule` defines when the backup job for harbor will run.
+              This should be set to run shortly after velero backups in wc, in order to ensure that images needed for velero backups are backed up in harbor.
```

Comment:
```suggestion
              `schedule` defines when the backup job for Harbor will run.
              This should be set to run shortly after Velero backups in WC, in order to ensure that images needed for Velero backups are backed up in Harbor.
```

Reply:
Does this not also depend on how long time the Velero backup takes to complete?

It would be nice if they were triggered in sequence rather than on two different schedules. Issue worthy?

Reply:
Yes, it does definitely depend on that. This mostly reflects that most velero backups should be done within 30 mins.
It would be very nice if that was triggered in sequence, but I'm not sure that it's worth making some controller that would be able to fix that. Especially since this is spread out across two different kubernetes clusters, harbor backups in sc and velero backups in wc

Reply:
Since we are so hesitant to create new issues to prevent flooding the backlog (which I totally understand) could we find some way of still keep "nice to haves" around? I think even if this isn't something we want to add to the current backlog this _is_ something we want to solve in the future, i.e. that when you create a backup of your cluster all images that are currently in use should also be backed up.

Reply:
i can bring that up to see what we can do :+1: